### PR TITLE
Add CI workflows to configure labels and label pull-requests

### PR DESCRIPTION
**Pull request by wmontwe**
## Description
Add two new workflows:
* d4l-ci-label-configuration.yml: maintains GitHub labels for this project -> see `.worklow/labels.yml`
* d4l-ci-pr-labeler.yml: GitHub Action to assign labels based on folders in the project -> see `.worklow/labeler.yml`

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  color: 008672
+
+# sample
+- name: sample-android
+  description: SampleApp Android
+  color: 4514b7
+- name: sample-jvm
+  description: SampleApp JVM
+  color: 4514b7
+
```
Comment (bitPogo) [inline .github/labels.yml:73]: This could be aligned now with the others, right?

Comment (wmontwe) [reply .github/labels.yml]: this is updated

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - buildSrc/**/*
+
+documentation:
+  - docs/**/*
+  - README.adoc
+  - LICENSE
+  - CHANGELOG.adoc
+  - CODE-OF-CONDUCT.adoc
+  - CONTRIBUTION.adoc
+  - readmeOSS.adoc
```
Comment (bitPogo) [inline .github/labeler.yml:14]: Maybe I overlook something here - shouldn’t be `readmeOSS.adoc` located at the root dir?

**Review by bitPogo — COMMENTED** [review]
Looks good to me - just one pending question before I approve it.

File: .github/labeler.yml (hunk 2 above)
Comment (bitPogo) [reply .github/labeler.yml:14]: Solved by: https://github.com/d4l-data4life/hc-fhir-sdk-kmp/pull/30

**Review by bitPogo — APPROVED** [review]
Good to go

**Pull request merged**